.github/workflows/qns.yml

```diff
@@ -213,15 +213,20 @@ jobs:
                 fi
                 [ -n "$RESULT" ] || continue
                 DIFF=$(wdiff -n "$BASELINE" - <<< "$RESULT" || true)
-                RESULT=$(echo "$DIFF" | sed -E "s/\[-|-\]/~~/g; s/\{\+|\+\}/\*\*/g")
+                RESULT=$(echo "$DIFF" | sed -E "s/\[-/ _~~/g; s/-\]/~~_ /g; s/\{\+/ <ins>\*\*/g; s/\+\}/\*\*<\/ins> /g")
```

Comment:
Do you need italics?  It renders pretty strangely for me.

Reply:
The strikethrough is hard to see on some letters, e.g., ~~A~~. But maybe it's my font?

Wondering if these would be better: ~~<ins>A</ins>~~ A ***A***